app/javascript/analytics/dashboard.js

```diff
@@ -45,15 +45,23 @@ function writeCards(data, timeRangeLabel) {
 }
 
 function drawChart({ id, showPoints = true, title, labels, datasets }) {
-  const chartOptions = showPoints
-    ? {}
-    : {
-        elements: {
-          point: {
-            radius: 0,
-          },
+  const chartOptions = {
+    elements: {
+      point: {
+        // The default is 3: https://www.chartjs.org/docs/latest/configuration/elements.html#point-configuration
+        radius: showPoints ? 3 : 0,
+      },
+    },
+    scales: {
+      y: {
+        type: 'linear',
+        suggestedMin: 0,
```

Comment:
```suggestion
       min: 0,
```
Maybe we should change this to the more prescriptive `min`? - `suggestedMin` works great when the data is more than just 0, but I noticed when the data is all zeroes, we still see a negative value in the axis:
![axis goes to minus 1](https://user-images.githubusercontent.com/20773163/166452660-2991bfa6-7276-4dae-b995-3710b684f4aa.png)

Reply:
@aitchiss Thanks. While I noticed it when I worked on this, this `drawChart` function looks generic and would take dataset arguments with negative integers in the future. Not sure what the original intention is #2037, but would be safe to me (who is new to Forem) if we keep this unchanged 😀

Reply:
That makes sense!

Reply:
note: this looks like a kind of bug to me. A further test bed is just prepared: https://codepen.io/tnir/pen/bGLdPOO

Reply:
note: this looks like a kind of bug to me. A further test bed is just prepared: https://codepen.io/tnir/pen/bGLdPOO